modules/auxiliary/gather/windows_secrets_dump.rb

```diff
@@ -1110,7 +1112,7 @@ def run
     end
     @winreg.close if @winreg
     @tree.disconnect! if @tree
-    simple.client.disconnect! if simple&.client&.is_a?(RubySMB::Client)
+    simple.client.disconnect! if simple&.client.is_a?(RubySMB::Client)
```

Comment:
Interesting change. I was in favor of leaving this as is, but considering that every object supports the `is_a?` function (inherited from the Object class), I can see why the safe navigation operator isn't needed _for this specific case_

Reply:
Marking as resolved, this was more a comment for anyone who was wondering why this change was made (like myself when I stumble across this code again 😄)